ops/framework.py

```diff
@@ -428,7 +501,7 @@ def __get__(self, emitter, emitter_type):
         return instance
 
     @classmethod
-    def define_event(cls, event_kind, event_type):
+    def define_event(cls, event_kind: str, event_type: 'Type[EventBase]'):
```

Comment:
FWIW pyright doesn't care what I put in the square brackets - it passes no matter what :P

Reply:
I think the type is correct as it is, _TypeVar is a variable and as such it would make sense to use it here only if it also occurred in the return signature.